ietf/templates/doc/shepherd_writeup.txt

```diff
@@ -63,30 +65,30 @@ to answer all of them.
    to be handed off to the responsible Area Director?
 
 10. Several IETF Areas have assembled [lists of common issues that their
-    reviewers encounter][6]. Do any such issues remain that would merit specific
-    attention from subsequent reviews?
+    reviewers encounter][6]. Are any such issues still present in the document?
 
-11. What type of RFC publication is being requested on the IETF stream (Best
-    Current Practice, Proposed Standard, Internet Standard, Informational,
-    Experimental, or Historic)? Why is this the proper type of RFC? Do all
-    Datatracker state attributes correctly reflect this intent?
+11. What type of RFC publication is being requested on the IETF stream ([Best
+    Current Practice][13], [Proposed Standard, Internet Standard][14],
+    [Informational, Experimental or Historic][15])? Why is this the proper type
+    of RFC? Do all Datatracker state attributes correctly reflect this intent?
 
-12. Has the interested community confirmed that any and all appropriate IPR
-    disclosures required by [BCP 78][7] and [BCP 79][8] have been filed? If not,
-    explain why. If yes, summarize any discussion and conclusion regarding the
-    intellectual property rights (IPR) disclosures, including links to relevant
-    emails.
+12. Have the authors, editors and contributors confirmed that any and all
+    appropriate IPR disclosures required by [BCP 78][7] and [BCP 79][8] have
+    been filed? If not, explain why. If yes, summarize any discussion and
+    conclusion regarding the intellectual property rights (IPR) disclosures,
+    including links to relevant emails.
```

Comment:
When I've doc shepherding, some non-small percentage of "no IPR" replies were private. This could be taken as implying such replies should be on the list.

Reply:
I think we need to have a position whether IPR replies need to be publicly recorded on the list first.
This text could then reflect that position.